cristin-biobank/build.gradle

```diff
@@ -22,7 +19,7 @@ tasks.named('test') {
     useJUnitPlatform {
         excludeTags 'karateTest'
     }
-    failFast = true
+//    failFast = true
```

Comment:
what does this do? remove completely?

Reply:
failfast stops execution after a failed test, I prefere to fix more than one bug at a time, if there are good reason to keep it I will add it back